crates/typed-store-derive/Cargo.toml

```diff
@@ -15,6 +15,7 @@ proc-macro = true
 proc-macro2 = "1.0.47"
 quote = "1.0.9"
 syn = { version = "1.0.102", features = ["derive"] }
+tokio = { version = "1.21.2", features = ["full", "test-util"] }
```

Comment:
please do not enable test-utils in normal builds.

Reply:
you can have another `tokio` line under the dev-deps section to enable this for testing purposes.